src/renderer/library/components/opds/BrowserResult.tsx

```diff
@@ -73,22 +73,34 @@ export class BrowserResult extends React.Component<IProps, undefined> {
                     !browserResult.data.navigation &&
                     !browserResult.data.groups) {
 
+                    const facetsRender = browserResult.data.facets?.map((facet, facetId) =>
+                        <section key={`facet-${facetId}`}>
+                            <br></br>
+                            <h3>{facet.title}</h3>
+                            <EntryList entries={facet.links}></EntryList>
+                        </section>,
+                    );
+
                     content = (
                         <>
-                            {
-                                browserResult.data.facets?.map((facet, facetId) =>
-                                    <section key={`facet-${facetId}`}>
-                                        <br></br>
-                                        <h3>{facet.title}</h3>
-                                        <EntryList entries={facet.links}></EntryList>
-                                    </section>,
-                                )
-                            }
-                            <EntryPublicationList
-                                opdsPublicationView={browserResult.data.publications}
-                                links={browserResult.data.links}
-                                pageInfo={browserResult.data.metadata}
-                            />
+                            <div className={Array.isArray(facetsRender) ? styles.publicationgrid : ""}>
+                                {
+                                    Array.isArray(facetsRender)
+                                        ? <div className={styles.publicationgriditem}>
```

Comment:
The CSS class `publicationgriditem` is missing from the style definition, is that intentional?
(see `src/renderer/assets/styles/opds.css` where only `publicationgrid` is defined)

Reply:
anticipation :)

Reply:
Okay, so I will test and merge as-is.